README.md

```diff
@@ -90,6 +90,7 @@ _(scroll right on table to see all 5 columns)_
 | Evernote                       | [joplinapp.org](https://joplinapp.org/)                                                                                                                                | [GitHub](https://github.com/laurent22/joplin)                                                                                              | JavaScript, TypeScript                                                         | ![GitHub Repo stars](https://flat.badgen.net/github/stars/laurent22/joplin)                                                                                                                      |
 | Facebook                       | [odinclone.herokuapp](https://odinclone.herokuapp.com/)                                                                                                                | [GitHub](https://github.com/lucaskenji/odinclone-app)                                                                                      | MERN                                                                           | ![GitHub Repo stars](https://flat.badgen.net/github/stars/lucaskenji/odinclone-app)                                                                                                              |
 | Facebook                       | [github.com (screenshot)](https://github.com/KristianWEB/fakebooker-frontend#screenshots)                                                                              | [GitHub](https://github.com/KristianWEB/fakebooker-frontend)                                                                               | react, graphql, mongodb                                                        | ![GitHub Repo stars](https://flat.badgen.net/github/stars/KristianWEB/fakebooker-frontend)                                                                                                       |
+| Facebook                       | [facebook-clone-dev.vercel.app](https://facebook-clone-dev.vercel.app/)                                                                                                | [GitHub](https://github.com/jhony-v/facebook-clone)                                                                                        | React, Recoil, Styled-component, Typescript, NextJs, Motion, StitchesJs        | ![GitHub Repo stars](https://flat.badgen.net/github/stars/jhony-v/facebook-clone)                                                                                                                |
```

Comment:
any kind of video and images are not visible and auth. doesn't working .
Looks like a frontend project .